examples/common.rs

```diff
@@ -0,0 +1,217 @@
+use std::{env, iter::repeat, thread, time, time::Duration};
+
+use crypto::{
+    aead::{AeadDecryptor, AeadEncryptor},
+    aes::KeySize::KeySize256,
+    aes_gcm::AesGcm,
+};
+use curv::{
+    arithmetic::traits::Converter,
+    elliptic::curves::traits::{ECPoint, ECScalar},
+    BigInt, FE, GE,
+};
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+
+pub type Key = String;
+
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct AEAD {
+    pub ciphertext: Vec<u8>,
+    pub tag: Vec<u8>,
+}
+
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct PartySignup {
+    pub number: u16,
+    pub uuid: String,
+}
+
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct Index {
+    pub key: Key,
+}
+
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct Entry {
+    pub key: Key,
+    pub value: String,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Params {
+    pub parties: String,
+    pub threshold: String,
+}
+
+#[allow(dead_code)]
+pub fn aes_encrypt(key: &[u8], plaintext: &[u8]) -> AEAD {
```

Comment:
is there a need to `#[allow(dead_code)]` for public code?
https://stackoverflow.com/a/53466873/2768367

Reply:
Added it since it warned even it's pub

Reply:
![image](https://user-images.githubusercontent.com/4182346/65387093-a4157c00-dd7e-11e9-84a4-157a4d45a33a.png)